packages/atomic-react/rollup.config.js

```diff
@@ -0,0 +1,40 @@
+import typescript from '@rollup/plugin-typescript';
+import commonjs from '@rollup/plugin-commonjs';
+import {nodeResolve} from '@rollup/plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+import nodePolyfills from 'rollup-plugin-polyfill-node';
+import {terser} from 'rollup-plugin-terser';
+
+const outputIIFE = ({minify}) => ({
+  file: `dist/iife/atomic-react${minify ? '.min' : ''}.js`,
+  format: 'iife',
```

Comment:
why exactly did UMD not work?

Reply:
We already have a a common js output produced by the TypeScript compiler.

I don't think we should remove/rename/modify it to UMD (ie: instead of being produced by TypeScript to common js, now using rollup to produce browser compatible UMD). 

The reason being that it does have an impact on various tools (bundler) down the line which *may* break in unexpected manner. Frontend tools are pretty finicky, after all ;)

So to stay on the safe and coupled with the fact that adding a umd (commonjs + iife) seems to overlap with the current common js output, I opted to ad an iife only.

Thoughts ?